### PR TITLE
Added Window to the text properties of ucrLog and ucrScript windows

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes#5252

@africanmathsinitiative/developers This is ready for review

### CODE REVIEW

**Comment by Wycklife** [comment]
@Muthenya Kindly review this

**Comment by Muthenya** [comment]
This looks good.

**Comment by Muthenya** [comment]
@rdstern  I'm just a bit curious. Is there need for the same edit to be made on the other windows, that is the **Metadata** and **Data View** windows? I don't have a logic reason for this, just asking.

**Comment by rdstern** [comment]
I was just requesting consistency of the name in the View Menu and the name once the Window was opened.  In the View Menu it is called Log Window, so let's have that name once opened too.  The other names were already consistent.

**Comment by Muthenya** [comment]
Okay. I understand now. This is fine then. Thanks.

**Comment by Muthenya** [comment]
@dannyparsons this is ready.

File: instat/ucrScript.Designer.vb [hunk 1]
```diff
@@ -180,12 +189,11 @@ Partial Class ucrScript
...
         Me.lblHeader.ForeColor = System.Drawing.SystemColors.Control
-        Me.lblHeader.Location = New System.Drawing.Point(4, 0)
-        Me.lblHeader.Margin = New System.Windows.Forms.Padding(4, 0, 4, 0)
+        Me.lblHeader.Location = New System.Drawing.Point(3, 0)
         Me.lblHeader.Name = "lblHeader"
-        Me.lblHeader.Size = New System.Drawing.Size(608, 31)
+        Me.lblHeader.Size = New System.Drawing.Size(405, 20)
         Me.lblHeader.TabIndex = 8
-        Me.lblHeader.Text = "Script"
+        Me.lblHeader.Text = "Script Windows"
```
Comment (dannyparsons) [inline instat/ucrScript.Designer.vb]: `Window` not `Windows`

**Comment by Wycklife** [comment]
@dannyparsons I have made the change

**Pull request merged**